darwin/future/core/client.py

```diff
@@ -245,3 +245,5 @@ def raise_for_darwin_exception(response: requests.Response) -> None:
         raise Unauthorized(response)
     if response.status_code == 404:
         raise NotFound(response)
+    if response.status_code == 422:
```

Comment:
We may want to surface / deal / pprint with 422s later, especially if because the endpoints generally do a good job of pointing out exactly **what** causes 422s